Date.js

```diff
@@ -0,0 +1,20 @@
+//creating date objects
+const d=new Date();
+console.log(d);//returns Sat Mar 19 2022 19:21:42 GMT+0530 (India Standard Time)
+
+const x = new Date(2018, 11, 24, 10, 33, 30, 0);
+console.log(x); // returns Mon Dec 24 2018 10:33:30 GMT+0530 (India Standard Time)
+
```

Comment:
try adding and subtracting the dates

Reply:
I done with adding and subtracting the dates.